src/test/java/org/takes/tk/TkSlf4jTest.java

```diff
@@ -55,4 +68,28 @@ public void logsException() throws IOException {
         new TkSlf4j(new TkFailure(new IOException(""))).act(new RqFake());
     }
 
+    /**
+     * {@link TkSlf4j} can output an empty body for {@link TkEmpty}.
+     * @throws IOException if some I/O problem occurred.
+     */
+    @Ignore
+    @Test
+    public void testEmptyResponseBody() throws IOException {
```

Comment:
@prondzyn The name of your test is not correct, please read [Test Method Names](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names)